### PR TITLE
Merge JS-16.1 into experimental-upstream-ironic-JS-16.1-00

### PR DESCRIPTION
**Pull request by rpioso**
This synchronizes branch `experimental-upstream-ironic-JS-16.1-00` with branch `JS-16.1`. All commits on `JS-16.1` are available on `experimental-upstream-ironic-16.1-00`.

The SHAs of the commits this adds to `experimental-upstream-ironic-JS-16.1-00` are identical to those on `JS-16.1`. That will later make it easier to merge `experimental-upstream-ironic-JS-16.1-00` into `JS-16.1`.

### CODE REVIEW

**Comment by rpioso** [comment]
Please use the `Merge pull request` option when closing this pull request, instead of `Squash and merge` or `Rebase and merge`.

TL;DR, that will cause the SHAs of the commits on the `dsp-jetpack` `JS-16.1` and `experimental-upstream-ironic-JS-16.1-00` branches to be identical through this point-in-time.

The `Merge pull request` option will retain the commits on the [`rpioso:experimental-upstream-ironic-JS-16.1-00`](https://github.com/rpioso/JetPack/tree/experimental-upstream-ironic-JS-16.1-00) `from` branch. Their SHAs will not be changed. Those commits were created in a local clone by merging `JS-16.1` and resolving it by fast-forward. No commit SHAs were changed.

Therefore, the end result is that the `dsp-jetpack` `experimental-upstream-ironic-JS-16.1-00` and `JS-16.1-00` branches will both include the commits this pull request contains. At this point-in-time, those two (2) branches will point at the same commit.

**Review by ajya — APPROVED** [review]

**Review by gaelrehault — APPROVED** [review]

**Comment by heydog15** [comment]
2 Approvals.  Go ahead and merge it
